iFixit/react-commerce
# Setup E2E testing with Cypress

This PR introduces E2E testing with Cypress.

- Cypress setup
- CI setup with Github actions
- Adds tests for the newsletter subscription form

## QA

- Verify that email subscription form is sufficiently tested. In particular the user should see an error when clicking subscribe and the email is missing or invalid, and he should see if the request has been successful or a server error occurred.
- Verify also that all tests are correctly processed by CI and have passed (see github action logs).
